self-tests.yml

```diff
@@ -250,6 +250,18 @@ functions:
 #######################################
 
 tasks:
+  - name: try-host
```

Comment:
probably don't commit this?

Reply:
Yeah, sorry, committed my working directory by mistake. Fixed.